pkg/deploy/lattice/service_manager.go

```diff
@@ -124,7 +125,62 @@ func svcStatusFromCreateSvcResp(resp *CreateSvcResp) ServiceInfo {
 	return svcInfo
 }
 
+func (m *defaultServiceManager) checkAndUpdateOwnership(ctx context.Context, tags services.Tags, svc *Service, arn string) error {
```

Comment:
I think ownership should be more abstract. Probably in cloud/lattice interface. I would think of following methods that should work together for different use-cases.

IsNotManaged(tags) bool (not managed at all)
IsManaged (tags) bool (managed by current controller)
tryOwn(arn) error (best effort of trying to acquire ownership)
handleOwnership(arn) error (current version of checkAndUpdateOwnership)

Reply:
Also in long run we would need to have ownership check on every write operation. To make it robust there should be a cache with known ownership. So I see it that ownership should stand aside/above of any of resource. If each resource will have own ownership pattern, then it should be interface.

Reply:
Will think of way to abstract this in the next rev, thanks